.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Remove package-lock.json
+package-lock.json
```

Comment:
We might need the lock file to be in the version control, this allow for some module binding and it always advisable to have it in version control